tests/cases/compiler/checkJsFiles_skipDiagnostics.ts

```diff
@@ -27,6 +27,18 @@ x(
 
 x();
 
+/* @ts-ignore */
+/*another comment
+ that could be multiline*/
+
+x();
+
+/* @ts-ignore
+continuing comment
+ that could be multiline*/
+
+x();
+

```

Comment:
@sheetalkamat sorry, I'm not clear on what you mean by this: https://github.com/microsoft/TypeScript/pull/38228#issuecomment-624215849

Is what you're looking for that a multiline comment containing a directive should disable the line below it?

If so: sure, I can change the scanner to allow for that. Would you like it to be just the first and last line? Or any line in the comment?